desktop version/index.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Phil's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap">
+</head>
+
+<body>
+    <!--Toolbar Section-->
+    <header>
+        <a href="" class="logo">
```

Comment:
While you've done a great deal at making this design, note that this project does not imply initializing an entirely new project within the current project, you're expected to utilize media queries to modify the styles in a way to be adaptable to desktop devices.

This is important because you typically would want to load the same project but in ways to adjust to the user's screen size. You may go through the [Responsive web design (RWD) with media queries](https://github.com/microverseinc/curriculum-html-css/blob/9994802d0d47e69b3dee918d3f32175fdead7ea3/responsivness.md) activity for more details.

You have created a new project (desktop version) within the current project. Kindly modify the already existing mobile version project using media queries to match the desktop design of your chosen template.